### ISSUE TITLE
deactivate_plugin - function does not exists

### ISSUE THREAD

**Issue by Chrico** [paulgibbs/behat-wordpress-extension#145]
I'm using this extension together with following context:

```php
use Behat\Behat\Context\SnippetAcceptingContext;
use PaulGibbs\WordpressBehatExtension\Context\RawWordpressContext;

/**
 * Define application features from the specific context.
 */
class FeatureContext extends RawWordpressContext implements SnippetAcceptingContext {
	// snip

	/**
	 * @Given /^The plugin "(?P<plugin>[^"]+)" is deactivated$/
	 */
	public function thePluginIsDeactivated( $plugin ) {

		$this->getDriver()
			->deactivatePlugin( $plugin );
	}
}
```

## Expected behaviour
The deactivation of the plugin works correctly with `default_driver: wpphp`.

## Current behaviour
When using the `default_driver: wpphp` (instead of `wpcli` where it is working), i see following output:

```
Fatal error: Call to undefined function PaulGibbs\WordpressBehatExtension\Driver\Element\Wpphp\deactivate_plugin() (Behat\Testwork\Call\Exception\FatalThrowableError)
 ```

## Possible solution
The `Driver\WpphpDriver::deactivatePlugin( $plugin )` is using internally `$this->plugin->deactivate($plugin);` which refers to the `Driver\Element\Wpphp\PluginElement::deactivate` which calls internally the `PluginElement::update`-method. This method runs `deactivate_plugin` which does not exists.

WordPress just provides a `activate_plugin`-function, but for deactivating plugins you've to call [`deactivate_plugins()`](https://github.com/WordPress/WordPress/blob/3609d0c4c50f41362f6be1c39bffc1875748e08f/wp-admin/includes/plugin.php#L652). The singular-one is not there.

## Your environment
*Version used:*
 ``` 
    "paulgibbs/behat-wordpress-extension": "^0.9.0",
    "behat/mink-goutte-driver": "^1.2",
    "behat/mink-selenium2-driver": "^1.3"
```

*Environment name and version: PHP 7.1.

**Comment by paulgibbs** [comment]
I am very sorry. Thank you for letting us know. After we get #144 merged in this/next week, I'll fix this, and make a new release.

Obviously, we don't have a feature test for this or many other driver functions yet, so that's the next short-term priority. Again, post-#144.

**Comment by paulgibbs** [comment]
Keeping this as a reminder to write a feature to test this. :(

**Comment by rvodden** [comment]
The askismet plugin is installed by the default wordpress installer. We could write a really quick feature which turns that on and off?

**Comment by paulgibbs** [comment]
This was fixed in d46fb33f21afe74e97e96a274e003a6274b78240 and tests added in f682f8f5fe6c5d407c62aa8896e18914e21f9288
